[2022-08-06 19:36:16,869] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 19:36:16,888] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 19:36:16,888] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:16,888] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:16,888] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:16,911] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 19:36:16,932] {standard_task_runner.py:52} INFO - Started process 371 to run task
[2022-08-06 19:36:16,937] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp5xyu32y_', '--error-file', '/tmp/tmp1dtjywaw']
[2022-08-06 19:36:16,938] {standard_task_runner.py:80} INFO - Job 90: Subtask second_task
[2022-08-06 19:36:17,026] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:17,128] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 19:36:17,130] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:17,130] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:17,142] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:17,144] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:17,144] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:17,175] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T193616, end_date=20220806T193617
[2022-08-06 19:36:17,228] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:17,263] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:11,269] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 21:50:11,284] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 21:50:11,284] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:11,284] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:11,284] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:11,302] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 21:50:11,308] {standard_task_runner.py:52} INFO - Started process 559 to run task
[2022-08-06 21:50:11,311] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp9_l70qy4', '--error-file', '/tmp/tmpw9s90reg']
[2022-08-06 21:50:11,311] {standard_task_runner.py:80} INFO - Job 91: Subtask second_task
[2022-08-06 21:50:11,383] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:11,518] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 21:50:11,519] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:11,520] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:11,542] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:11,544] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:11,544] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:11,592] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T215011, end_date=20220806T215011
[2022-08-06 21:50:11,642] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:11,688] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:55:42,113] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 21:55:42,136] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 21:55:42,137] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:42,137] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:55:42,137] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:42,157] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 21:55:42,171] {standard_task_runner.py:52} INFO - Started process 358 to run task
[2022-08-06 21:55:42,175] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwl7op7l5', '--error-file', '/tmp/tmpla2no0ac']
[2022-08-06 21:55:42,175] {standard_task_runner.py:80} INFO - Job 92: Subtask second_task
[2022-08-06 21:55:42,279] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:55:42,387] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 21:55:42,388] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:55:42,388] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:55:42,404] {subprocess.py:85} INFO - Output:
[2022-08-06 21:55:42,406] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:55:42,407] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:55:42,447] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T215542, end_date=20220806T215542
[2022-08-06 21:55:42,481] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:55:42,561] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:10:41,212] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:10:41,226] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:10:41,226] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:41,227] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:10:41,227] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:41,250] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 22:10:41,256] {standard_task_runner.py:52} INFO - Started process 365 to run task
[2022-08-06 22:10:41,259] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmlf1mg_e', '--error-file', '/tmp/tmpzdltukwc']
[2022-08-06 22:10:41,260] {standard_task_runner.py:80} INFO - Job 90: Subtask second_task
[2022-08-06 22:10:41,329] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:10:41,410] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 22:10:41,411] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:10:41,412] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:10:41,426] {subprocess.py:85} INFO - Output:
[2022-08-06 22:10:41,428] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:10:41,429] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:10:41,462] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T221041, end_date=20220806T221041
[2022-08-06 22:10:41,511] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:10:41,545] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:18:29,374] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:18:29,395] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:18:29,395] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:29,395] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:18:29,395] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:29,413] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 22:18:29,420] {standard_task_runner.py:52} INFO - Started process 375 to run task
[2022-08-06 22:18:29,423] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpfhj65nlk', '--error-file', '/tmp/tmp9tfecqxe']
[2022-08-06 22:18:29,423] {standard_task_runner.py:80} INFO - Job 91: Subtask second_task
[2022-08-06 22:18:29,500] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:18:29,643] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 22:18:29,644] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:18:29,645] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:18:29,662] {subprocess.py:85} INFO - Output:
[2022-08-06 22:18:29,665] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:18:29,665] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:18:29,702] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T221829, end_date=20220806T221829
[2022-08-06 22:18:29,755] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:18:29,792] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:02,119] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:22:02,133] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [queued]>
[2022-08-06 22:22:02,133] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:02,133] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:02,133] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:02,149] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-27 00:00:00+00:00
[2022-08-06 22:22:02,156] {standard_task_runner.py:52} INFO - Started process 410 to run task
[2022-08-06 22:22:02,159] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-27T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpl8tfx7en', '--error-file', '/tmp/tmprwjbu78t']
[2022-08-06 22:22:02,160] {standard_task_runner.py:80} INFO - Job 90: Subtask second_task
[2022-08-06 22:22:02,267] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-27T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:02,361] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-27T00:00:00+00:00
[2022-08-06 22:22:02,362] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:02,362] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:02,376] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:02,378] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:02,378] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:02,420] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210827T000000, start_date=20220806T222202, end_date=20220806T222202
[2022-08-06 22:22:02,450] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:02,483] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
